### PR TITLE
Update index.php

### PR DESCRIPTION
**Pull request by dilipchauhan14**
Removed Useless Lines.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -74,9 +75,6 @@
  
  }
 
-function DoNothing(){
```
Comment (dilipchauhan14) [inline index.php]: Useless Function

File: index.php [hunk 2]
```diff
@@ -250,10 +248,6 @@ function test_input($data) {
         session_destroy();
     }
 ?>
-shdgkhdjsdsdadjq
```
Comment (dilipchauhan14) [inline index.php]: Useless Lines

File: index.php [hunk 3]
```diff
@@ -1,4 +1,5 @@
-
```
Comment (dilipchauhan14) [inline index.php]: jsdjjf

File: index.php [hunk 4]
```diff
@@ -1,4 +1,5 @@
...
+<!-- Author: Sammarth Kapse -->
```
Comment (dilipchauhan14) [inline index.php]: Added Author

File: index.php [hunk 5]
```diff
@@ -1,4 +1,5 @@
...
+<!-- Date: 17/01/2020 -->
```
Comment (sammarth-kapse) [inline index.php]: Good Work.

**Pull request merged**